feature-libs/quote/components/header/order-summary/quote-header-order-summary.module.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import {
+  AuthGuard,
+  CmsConfig,
+  I18nModule,
+  provideDefaultConfig,
+} from '@spartacus/core';
+import { QuoteHeaderOrderSummaryComponent } from './quote-header-order-summary.component';
+import { QuoteHeaderPriceModule } from '../price/quote-header-price.module';
+import { QuoteHeaderSellerEditModule } from '../seller-edit/quote-header-seller-edit.module';
+import { QuoteActionsByRoleModule } from '../../actions/by-role/quote-actions-by-role.module';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    I18nModule,
+    QuoteHeaderPriceModule,
+    QuoteHeaderSellerEditModule,
+    QuoteActionsByRoleModule,
+  ],
+  providers: [
+    provideDefaultConfig(<CmsConfig>{
+      cmsComponents: {
+        QuoteHeaderOrderSummaryComponent: {
+          component: QuoteHeaderOrderSummaryComponent,
+          guards: [AuthGuard],
+        },
+      },
+    }),
+  ],
+  declarations: [QuoteHeaderOrderSummaryComponent],
+  exports: [QuoteHeaderOrderSummaryComponent],
+})
+export class QuoteHeaderOrderSummaryModule {}
```

Comment:
Suggestion for better organising the new component and its child ones:
- Name ist QuoteSummary component
- Have it in folder components / summary
- Have child components also in components / summary